### PR TITLE
update for 17.0

### PR DESCRIPTION
**Pull request by OnGle**

### CODE REVIEW

File: plan/main [hunk 1]
```diff
@@ -1,11 +1,11 @@
 #include <turnkey/base>
-#include <turnkey/lamp>
+#include <turnkey/lamp81>
```
Comment (JedMeister) [inline plan/main:2]: Did you forget to do a `common` PR for this new plan? It'd be easy for me to create one myself, but if you still have it handy, could you please open a new PR for this? Cheers man.

Comment (OnGle) [reply plan/main:2]: Spot on, I've added this in common, along with dynamic pins, see below.

**Comment by OnGle** [comment]
requires: https://github.com/turnkeylinux/common/pull/219 (missing 8.1 plan)
requires: https://github.com/turnkeylinux/common/pull/218 (for newer libpcre2)

**Comment by JedMeister** [comment]
Excellent. Looks good.

**Pull request merged**